### DISCUSSION TITLE
Retry output as debug or warn

### DISCUSSION THREAD

**Discussion by chb0github** [go-resty/resty#554] [Ideas]
The [code for output](https://github.com/go-resty/resty/blob/master/request.go#L771) on a retry attempt is currently an error - it's debatable if it's actually an error or not. In my case, retry is pretty typical and only when all attempts have been tried is it an error. 

I propose that this code

```go
			resp, err = r.client.execute(r)
			if err != nil {
				r.client.log.Errorf("%v, Attempt %v", err, r.Attempt)
			}
```
Be changed to 
```go
r.client.log.Warnf
```
Or
```go
r.client.log.Debugf
```
So that it can be suppressed unless there is an issue and that when all attempts are tried, then it outputs an `Errorf`

If this is ok, I will make the change and submit a PR

**Comment by jeevatkm** [comment]
It has been implemented in v2.8.0
